SampleBrowser/SampleBrowser/UI/RootViewController.swift

```diff
@@ -35,6 +36,7 @@ class RootViewController: UIViewController,
         self.browserVC = BrowserViewController(engineProvider: engineProvider)
         self.searchVC = SearchViewController()
         self.themeManager = themeManager
+        self.currentWindowUUID = WindowUUID.unavailable
```

Comment:
We shouldn't be defaulting to `unavailable`, that's not a UUID that should ever be used on an actual window. I think we can probably just change this to generate a random `UUID()` for now.

```suggestion
        self.currentWindowUUID = UUID()
```